README.md

```diff
@@ -180,6 +181,22 @@ Override schema:
     tag: string      # The image tag to use
 ```
 
+#### Overriding Remote Tag using CLI
+
+Additionally, you can run a docker image with a specific remote tag using the CLI. An example of doing so looks like this:
+
+```
+sso tb up venue-config-service -t venue-config-service:my_branch
+```
+
+You can also override a remote tag when running a playlist. To override multiple remote tags for multiple services, you will need to provide a comma-separated list of service:tag arguments.
+
+```
+sso tb up -p vcs-deps -t postgres:my_branch1,redis:my_branch2
+```
+
+NOTE: ECR does not support forward slashes (/) in an image tag, so task/ACC-1234/chore will be normalized to task-ACC-1234-chore when the image is pushed to ECR.
```

Comment:
This is touchbistro specific stuff, tb is a more general tool. I would document that slash gotcha in tb-registry